client/src/redux/reducers/index.js

```diff
@@ -15,7 +15,7 @@ import authenticatedUser from './authenticated_user.reducer';
 import { pushKeys } from './systems.reducers';
 import notifications from './notifications.reducers';
 import workbench from './workbench.reducers';
-import welcomeMessages from './welcome.reducers';
+import introMessages from './intro.reducers';
```

Comment:
This references [a file](https://github.com/TACC/Core-Portal/blob/826495e/client/src/redux/reducers/welcome.reducers.js) that likely needs to be renamed.